modules/images/image-loading-optimization/detection/detect.js

```diff
@@ -5,6 +5,43 @@ const doc = win.document;
 
 const consoleLogPrefix = '[Image Loading Optimization]';
 
+const storageLockTimeSessionKey = 'iloStorageLockTime';
+
+/**
+ * Checks whether storage is locked.
+ *
+ * @param {number} currentTime    - Current time in milliseconds.
+ * @param {number} storageLockTTL - Storage lock TTL in seconds.
+ * @return {boolean} Whether storage is locked.
+ */
+function isStorageLocked( currentTime, storageLockTTL ) {
+	try {
```

Comment:
Shouldn't we include a check for when the TTL is 0 here? Something like:
```suggestion
function isStorageLocked( currentTime, storageLockTTL ) {
	if ( ! storageLockTTL ) {
		return false;
	}

	try {
```

Reply:
Yeah, if `storageLockTTL <= 0`.